pkg/evaluator/evaluator_test.go

```diff
@@ -344,6 +398,26 @@ a := {name:"fox" age:39+(three)}`: "{name:fox age:42}",
 	}
 }
 
+func TestDot(t *testing.T) {
+	tests := map[string]string{
+		// m := {name: "Greta"}
+		"print m.name":    "Greta",
+		`print m["name"]`: "Greta",
+		`s := "name"
+		print m[s]`: "Greta",
```

Comment:
Should there be a test case for looking up a key not in the map? I can't remember what the semantics of that is? Returns zero value like Go? (as an aside, what's the zero value of any? doesn't seem to be in the spec)

ok, I see from below value.go that a bad index is an error. leaving this comment for the any question.

:pineapple:

Reply:
ok, good point. I've added a test case. Will add more when implementing `has` and `del` functions.

I wonder if I should return the zero value....

Reply:
sorry, no good at not answering.